src/Core/Grid/Position/RowUpdateInterface.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Grid\Position;
+
+/**
+ * Interface RowUpdateInterface contains the modification for a
+ * designated row.
+ */
+interface RowUpdateInterface
```

Comment:
i would suggest naming it `ModifierRowInterface` or similar as it's a noun and not a verb, wdyt? same for implementation and collection.

Reply:
Update can be a noun as well, that's how I pictured it
But I see your point there can be a misunderstanding, what do you think of `RowModificationInterface`